src/appmixer/intelswift/core/FindConversation/FindConversation.js

```
@@ -0,0 +1,21 @@
+'use strict';
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { tenantId, projectUUID, lastDays = 1, properties = "origin, contactId, assignees, assistantId, segment, sentiment, status, humanHandOff, resolvedAt, resolvedBy" } = context.messages.in.content;
+
+        let body = {};
```

ESLint

[error] 'body' is assigned a value but never used.

(no-unused-vars)